lib/jsdom/browser/Window.js

```diff
@@ -8,6 +8,9 @@ const { installInterfaces } = require("../living/interfaces");
 const { define, mixin } = require("../utils");
 const Element = require("../living/generated/Element");
 const EventTarget = require("../living/generated/EventTarget");
+const EventHandlerNonNull = require("../living/generated/EventHandlerNonNull.js");
+const OnBeforeUnloadEventHandlerNonNull = require("../living/generated/OnBeforeUnloadEventHandlerNonNull.js");
+const OnErrorEventHandlerNonNull = require("../living/generated/OnErrorEventHandlerNonNull.js");
```

Comment:
Nit: although I like the .js, it'd be better to be consistent with the nearby requires, which don't have it.

I'd support a separate PR that integrates https://www.npmjs.com/package/eslint-plugin-node and enforces extensions throughout the project.

Reply:
This nit still remains

Reply:
It’s [eslint‑plugin‑import](https://www.npmjs.com/package/eslint-plugin-import) that enforces extensions in `import`s and `require(…)`s.